include/distconfig.php

```diff
@@ -418,6 +419,7 @@
 // This will look for README.md file on the path and render it.
 // The name of "README.md" isn't configurable for now to simply follow GitHub's conventions.
 
+// $config->setParsedownPath('/usr/share/php/Parsedown/Parsedown.php'); // optional. Use if you have Parsedown installed without PEAR/Composer
```

Comment:
This looks wrong, only the path, not the script path

Reply:
Fixed in [7c42b89](https://github.com/websvnphp/websvn/pull/202/commits/7c42b89242efc45a5d7ac506fd5acfc66e0b0f06)